src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java

```diff
@@ -87,14 +87,20 @@ public Ride postRide(
         @Parameter(name="pickupLocation", description="String, Location the ride starts", example="Phelps Hall", required = true)
         @RequestParam String pickupLocation,
 
+        @Parameter(name="pickupRoom", description="String, Room number for the pickupLocation", example="1160", required = true)
+        @RequestParam String pickupRoom,    //ADDED
```

Comment:
Remove this and other related comments.

Reply:
I'll remove them ni the next PR, where this entire EPIC will be resolved. For now, I'd like to keep them so that I know what is changed and what not

Reply:
These comments are not helpful to readers and shouldn't be committed in the first place. Use git blame or look at the diff of the commits if you want to know what you changed.

Reply:
I've deleted those comments accordingly. Please give it a look now.